terraform_scripts/beanstalk_env_vars.php

```diff
@@ -0,0 +1,40 @@
+<?php
+$usage = "Usage: php beanstalk_env_vars.php <json_file.json>\n";
+if ($argc < 2) {
+    echo $usage;
+    exit(1);
+}
+
+$jsonFile = $argv[1];
+if (! file_exists($jsonFile)) {
+    printf("Cannot reach file %s", $jsonFile);
+    exit(1);
+}
+
+$text = '
+  setting {
+    namespace = "aws:elasticbeanstalk:application:environment"
+    name      = "{{name}}"
+    value     = "{{value}}"
+  }
+';
+$contents = file_get_contents($jsonFile);
+$jsonData = json_decode($contents, true);
+if (false === $jsonData) {
+    printf("Invalid json data: %s", json_last_error_msg());
+    exit(1);
+}
+
+print_r($jsonData);
+
+$result = [];
+foreach ($jsonData as $key => $value) {
+    // why doesn't php have named arguments yet =(
```

Comment:
i think you mean python's string "format" function with variable substition or whatevers